elixir/APIRequest/README.md

```diff
@@ -0,0 +1,9 @@
+# `APIRequest` Application
+
+Welcome to the `APIRequest` AtomVM application.
+
+The `APIRequest` AtomVM application connects to a wifi, and when the connection is established retrieves and prints on console a GET request over https.
```

Comment:
Maybe we could change this to "... prints the result on the console...". Reading this I half expected the ESP32 to be the server getting the API request from a client, and printing the actual request on a the console. 

That would be a strange example, but there were enough details left out to make it hard to be certain which details were missing, especially for non-native English speakers. ;-)